server/game/cards/05-LoCR/TywinLannister.js

```diff
@@ -4,17 +4,22 @@ class TywinLannister extends DrawCard {
     setupCardAbilities() {
         this.interrupt({
             when: {
-                onCardDiscarded: (event) => this.exactlyOneCardDiscardedFromPlayersDeck(event)
+                onCardDiscarded: {
+                    aggregateBy: (event) => ({
+                        controller: event.card.controller,
```

Comment:
I believe so yeah.

Ideally I'd love to clean up when "cardStateWhenX" should be used & when "card" is to be used, as it can be confusing for Reactions vs Interrupts. Maybe a future update!